legacy/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>tds-common-legacy</artifactId>
+    <name>tds-common-legacy</name>
+
+    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.opentestsystem.common</groupId>
+        <artifactId>tds-common-parent</artifactId>
+        <version>0.0.2-SNAPSHOT</version>
```

Comment:
Notice that the parent POM dependency versions were also bumped to 0.0.2-SNAPSHOT by the release process.